src/redux/books/books.js

```diff
@@ -0,0 +1,14 @@
+import React from 'react';
+
+const BooksPage = () => (
+  <>
+    <div className="books-container d-flex">
+      <h2>Book List</h2>
+      <button type="button">Remove</button>
+    </div>
+    <input type="text" name="addBooks" placeholder="Book Title" />
+    <button type="button">Add Book</button>
```

Comment:
- [ ] Let's create `CreateNewBook` component to render this form. New book created will be received in `BooksPage` as `props`. This is project requirement and best practice.
- [ ] To have a good project structure I recommend you to put these components in `components` folder instead of `redux` folder.